packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -205,6 +205,24 @@ class WebViewController {
     return _channel.invokeMethod('loadUrl', url);
   }
 
+  Future<bool> get canGoBack async {
```

Comment:
We need Dartdoc for all the new public APIs.

Reply:
All right. b1b4d6c